misc/dbt-materialize/dbt/adapters/materialize/connections.py

```diff
@@ -51,6 +54,10 @@ def open(cls, connection):
                 f"  Supported versions: {SUPPORTED_MATERIALIZE_VERSIONS}"
             )
 
+        creds = connection.credentials
```

Comment:
dbt-snowflake uses the cluster in the actual connection string, but AFAIK this is not possible with Materialize cloud right now, so we set the cluster here instead.